docs/userguide.md

```diff
@@ -687,6 +688,23 @@ spec:
           op: "remove"
 ```
 
+### Overriding retained fields
+
+When computing the form of a managed resource that should appear in a cluster
+registered with KubeFed, the following operations are executed in sequence:
+
+ - A new resource is computed from the template of the federated resource
+ - If an existing resource is present, field values are retained
```

Comment:
`field values` are retained. I think it is not accurate because only partial field values are retained.

Reply:
Updated.  PTAL.